spec/views/projects/index.html.erb_spec.rb

```diff
@@ -13,4 +13,28 @@
   it 'renders a list of projects' do
     render
   end
-end
+
+  it 'shows all details about a project' do
+    chair = FactoryGirl.create(:chair)
+    project = FactoryGirl.create(:project, chair_id: chair.id)
+
+    project.update(status:true)
+    project.update(public:true)
+
+    visit projects_path
+
+    expect(page).to have_content(project.title)
+    expect(page).to have_content(l(project.created_at))
+    expect(page).to have_content(chair.name)
+
+    expect(page).to have_content(I18n.t('.public', default:'public'))
```

Comment:
geht das hier? Ich dachte, dass du hier sicher einen Pfad zu der Übersetzung von public angeben musst & das default in dem Fall nicht gebraucht wird